docs/style.css

```diff
@@ -1,3 +1,29 @@
+:root {
+    /* Nord Color Theme: https://www.nordtheme.com/
+       Details about the palette and use cause of each color: https://www.nordtheme.com/docs/colors-and-palettes */
+
+    /* Polar Night  */
+    --nord0: #2e3440;
+    --nord1: #3b4252;
+    --nord2: #434c5e;
+    --nord3: #4c566a;
+    /* Snow Storm */
+    --nord4: #d8dee9;
+    --nord5: #e5e9f0;
+    --nord6: #eceff4;
+    /* Frost */
+    --nord7: #8fbcbb;
+    --nord8: #88c0d0;
+    --nord9: #81a1c1;
+    --nord10: #5e81ac;
+    /* Aurora */
+    --nord11: #bf616a;
+    --nord12: #d08770;
+    --nord13: #ebcb8b;
+    --nord14: #a3be8c;
+    --nord15: #b48ead;
+}
+
```

Comment:
I suppose in an ideal world, just these color names would be set depending on dark-mode or light-mode. Everything else could then just refer to names instead of concrete colors